md/deploy-bundle.md

```diff
@@ -75,11 +75,11 @@ use the driver provided by your RDBMS vendor
 ## Wildfly installation
 
 ### Install Bonita BPM Platform in Wildfly
-Copy all files and directories from DEPLOY_ZIP_HOME/wildfly-10.1.0.Final to your Wildfly root directory (WILDFLY_HOME).
+Copy all files and directories from DEPLOY_ZIP_HOME/wildfly-10.1.0.Final/server to your Wildfly root directory (WILDFLY_HOME).
```

Comment:
Well, if you download wildfly-10.1.0.Final.zip, you don't have such 'server' directory, I'm afraid:
`$ unzip -l wildfly-10.1.0.Final.zip | grep server
        0  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/jboss/as/server/
        0  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/jboss/as/server/main/
        0  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/jboss/as/webservices/server/
        0  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/jboss/as/webservices/server/integration/
        0  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/jboss/as/webservices/server/integration/main/
        0  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/jboss/ws/cxf/jbossws-cxf-server/
        0  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/jboss/ws/cxf/jbossws-cxf-server/main/
        0  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/wildfly/clustering/server/
        0  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/wildfly/clustering/server/main/
     4752  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/jboss/as/server/main/module.xml
  1284707  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/jboss/as/server/main/wildfly-server-2.2.0.Final.jar
   150189  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/apache/activemq/artemis/main/artemis-jms-server-1.1.0.wildfly-017.jar
  1229360  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/apache/activemq/artemis/main/artemis-server-1.1.0.wildfly-017.jar
   114585  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/jboss/ws/cxf/jbossws-cxf-server/main/jbossws-cxf-server-5.1.5.Final.jar
     3089  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/jboss/ws/cxf/jbossws-cxf-server/main/module.xml
     2395  2016-08-18 19:12   wildfly-10.1.0.Final/domain/configuration/default-server-logging.properties
   329936  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/jboss/as/webservices/main/wildfly-webservices-server-integration-10.1.0.Final.jar
     2649  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/wildfly/clustering/server/main/module.xml
   160995  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/wildfly/clustering/server/main/wildfly-clustering-server-10.1.0.Final.jar
     4177  2016-08-18 19:12   wildfly-10.1.0.Final/modules/system/layers/base/org/jboss/as/webservices/server/integration/main/module.xml
`